### PR TITLE
refactor: change peer seeds and dns to per network

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Change peer_seeds and dns settings to be per network.

How Has This Been Tested?
---
npm test on some of the tests

### CODE REVIEW

File: common/config/presets/common.toml [hunk 1]
```diff
@@ -102,3 +68,37 @@ rpc_max_simultaneous_sessions = 10000
...
+    "981cc8cd1e4fe2f99ea1bd3e0ab1e7821ca0bfab336a4967cfec053fee86254c::/onion3/7hxpnxrxycdfevirddau7ybofwedaamjrg2ijm57k2kevh5q46ixamid:18141", #seoul
+    "f2ce179fb733725961a5f7e1e45dacdd443dd43ba6237438d6abe344fb717058::/onion3/nvgdmjf4wucgatz7vemzvi2u4sw5o4gyzwuikagpepoj4w7mkii47zid:18141", #stockholm
+    "909c0160f4d8e815aba5c2bbccfcceb448877e7b38759fb160f3e9494484d515::/onion3/qw5uxv533sqdn2qoncfyqo35dgecy4rt4x27rexi2her6q6pcpxbm4qd:18141", #sydney
+    # igor
+    "8e7eb81e512f3d6347bf9b1ca9cd67d2c8e29f2836fc5bd608206505cc72af34::/onion3/l4wouomx42nezhzexjdzfh7pcou5l7df24ggmwgekuih7tkv2rsaokqd:18141",
+    "00b35047a341401bcd336b2a3d564280a72f6dc72ec4c739d30c502acce4e803::/onion3/ojhxd7z6ga7qrvjlr3px66u7eiwasmffnuklscbh5o7g6wrbysj45vid:18141",
+    "40a9d8573745072534bce7d0ecafe882b1c79570375a69841c08a98dee9ecb5f::/onion3/io37fylc2pupg4cte4siqlsmuszkeythgjsxs2i3prm6jyz2dtophaad:18141",
+    "126c7ee64f71aca36398b977dd31fbbe9f9dad615df96473fb655bef5709c540::/onion3/6ilmgndocop7ybgmcvivbdsetzr5ggj4hhsivievoa2dx2b43wqlrlid:18141",
```
Comment (delta1) [inline common/config/presets/common.toml]: these should be moved into `[common.igor]` please

File: common/config/presets/common.toml [hunk 2]
```diff
@@ -102,3 +68,37 @@ rpc_max_simultaneous_sessions = 10000
...
+# DNS seeds
+# The DNS records in these hostnames should provide TXT records as per https://github.com/tari-project/tari/pull/2319
+# Enter a domain name for the TXT records: seeds.tari.com
+dns_seeds =["seeds.weatherwax.tari.com"]
+# The name server used to resolve DNS seeds  (Default: "1.1.1.1:53")
+# dns_seeds_name_server = "1.1.1.1:53"
+# Servers addresses, majority of them have to agree.
+# autoupdate_dns_hosts = [#server1, #server2, ...]
+# Set to true to only accept DNS records that pass DNSSEC validation (Default: true)
+dns_seeds_use_dnssec = false
```
Comment (delta1) [inline common/config/presets/common.toml]: `[common.igor]` should also have this but specify `seeds.igor.tari.com` instead

File: common/config/presets/common.toml (hunk 1 above)
Comment (Cifko) [reply common/config/presets/common.toml]: done

File: common/config/presets/common.toml (hunk 2 above)
Comment (Cifko) [reply common/config/presets/common.toml]: done

**Review by delta1 — APPROVED** [review]
LGTM but haven't tested it

**Review by stringhandler — APPROVED** [review]

**Comment by stringhandler** [comment]
Tests need updating, but happy otherwise

**Comment by stringhandler** [comment]
Closing in favour of other config changes

**Pull request closed without merging**